microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,24 +9,112 @@ ms.prod: microsoft-edge
 keywords: microsoft edge, compatibility, web platform, user agent string, ua string, ua overrides
 ---
 
-# Microsoft Edge User Agent String (Desktop)  
+# Retrieving User Agent Data in Microsoft Edge 
 
-A user agent \(UA\) string is able to be used to detect what version of a specific browser is being used on a certain operating system.  Like other browsers, Microsoft Edge includes this information in the `User-Agent` HTTP header whenever it makes a request to a site.  It may also be accessed via JavaScript by querying the value of `navigator.userAgent`.  
+The User Agent \(UA\) can be used detect what version of a specific browser is being used on a certain operating system. Microsoft Edge offers two mechanisms for retrieving User Agent information: via User Agent Client Hints (recommended) and the legacy User Agent string. These can be accessed as follows: 
+
+**User Agent Client Hints (Recommended)**
+-	*Server-side*: UA Client Hints are sent by default with the `Sec-CH-UA` `HTTPS` header 

```

Comment:
Can we call out "low entropy" vs "high entropy" hints. e.g. something like:

The "low entropy" hints, which contain a list of browser brands, major browser versions, and whether or not the device should receive mobile UX, are sent by default via the `Sec-CH-UA` and `Sec-CH-UA-Mobile` headers. Guidance for accessing additional information is provided below.

Reply:
Included note